QWeb/internal/table.py

```diff
@@ -166,7 +166,7 @@ def get_using_text_in_coordinates(self, coordinates: str, anchor: str, **kwargs)
         column: Optional[int]
         row_elem = None
         cell = None
-        locator = coordinates.split("/")
+        locator = coordinates.rsplit("/", 1)
```

Comment:
While this solves the immediate need for using '/' (f-slash) chars in text finders for cells this is not a proper solution.

Consider this kind of a table
```
|       | 2022/2023 | 2023/2024 |
---------------------------------
| 13/09 |    A1     |     B1    |
---------------------------------
| 24/12 |    A2     |     B2    |
```

If user wants to search for columns which have f-slashes then the user needs to know and understand that the r?X/c?Y can be flipped from the locator.
`Click Cell    c?13/09/2024/r2`

User could also use f-slash searches for both row and col which won't work with this
`Click Cell    r?2023/2024/c?13/09`


F-slash handling should be more robust and probably requires having an escape character and parsing rules for it.

Reply:
`Click Cell    r?2023/2024/c?13/09`

I don't think this is accepted format anyway? Why would they be finding via text search knowing both row and column? This is not in any examples in our documentation afaik. It's either r? or c?, right?

But true, I did not take into account that column can be given first...or can it? Never seen it used like this.

Reply:
Tested briefly, and as far as I can tell the other way around has never worked (Click Cell    c?13/09/2024/r2) and also using text search in both row and column has never been supported. This would be a new feature and out of the scope of this PR.

Reply:
meaning it has never worked even for values without "/" in them, for example:
`ClickCell            c3/r?Jim`

Reply:
Getting the cross section cell with text searches is a valid scenario if you don't want to hard code indices or if the ordering of the table is not guaranteed.

Using text search on both row and col seems to work at least with this trivial example at _table.html_
```
UseTable      Sample
GetCellText  r?Jill/c?Last
--> 'Smith'
```

![image](https://github.com/user-attachments/assets/94aa6b09-1426-4847-a1d1-a65af5048757)

I misremembered that you would be able to flip the cells but that alleviates the problem with this solution proposal, you could only look for rows which contain f-slashes and not columns.


Lets deal with f-slashes in a separate PR?

Reply:
Wouldn't affect the example given above, but removed with new commit. Added also note to kw docs that "/" is not supported.